extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -123,9 +123,23 @@ public class VertxHttpRecorder {
 
     public static final String MAX_REQUEST_SIZE_KEY = "io.quarkus.max-request-size";
 
-    // We do not use Integer.MAX on purpose to allow advanced users to register a route AFTER the default route
+    /**
+     * Order mark for route with priority over the default route.
+     * Don't use it directly, instead add a modulo 100 offset.
+     */
+    public static final int BEFORE_DEFAULT_ROUTE_ORDER_MARK = 1_000;
+
+    /**
+     * Default route order (i.e Static Resources, Servlet)
+     */
     public static final int DEFAULT_ROUTE_ORDER = 10_000;
 
+    /**
+     * Order mark for route without priority over the default route
```

Comment:
So, there are served after the static resources?

Reply:
yes, if there is no corresponding static resource (default route)